src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs

```diff
@@ -28,7 +28,7 @@ public OrasClient(string imageReference)
 
         public async Task PullImageAsync(string outputFolder)
         {
-            string command = $"pull  {_imageReference} -o {outputFolder}";
+            string command = $"pull  {_imageReference} -o \"{outputFolder}\"";
```

Comment:
Is it safer to add "" to the image reference?

Reply:
Thanks, fixed it.